src/context/FFContext.tsx

```diff
@@ -49,13 +56,19 @@ export const FFContextProvider: FC<FFContextProviderProps> = ({
   fallback = <p>Loading...</p>,
   async = false,
   initialEvaluations,
-  onError = () => void 0
+  onError = () => void 0,
+  onFlagNotFound = () => void 0
 }) => {
   const [loading, setLoading] = useState<boolean>(true)
   const [flags, setFlags] = useState<FFContextValue['flags']>({})
   const [clientInstance, setClientInstance] =
     useState<FFContextValue['client']>()
 
+// Use a reference to keep track of the latest loading state, allowing access to its current value
```

Comment:
I tried using the `loading` state and passing it to the new `onFlagNotFound` option. But it would still be `true` when the SDK initialised and `onFlagNotFound` was called.  I think this is due to the state being updated async, so this is what I went with as an alternative.